File: cifrado/escritorio/codigo/Kotlin-Fx-Tornado/prueba2/src/main/kotlin/com/example/demo/app/gestor.kt
package com.example.demo.app

class Gestor
{
    var palabra = "PRUEBA DE COHESION"
    var cesar = Cesar()
    var reverse = Reverse()
    var reverse_g = ReverseG()

    fun cifrar(palab:String)
    {
        cesar.palabra = palab
        reverse.palabra = palab
        reverse_g.palabra = palab

        cesar.cifrar()
        reverse.cifrar()
        reverse_g.cifrar()
    }

    fun cifradoSinActualizacion()
    {
        cesar.cifrar()
        reverse.cifrar()
        reverse_g.cifrar()
    }
    
}

File: cifrado/escritorio/codigo/Ruby-Fx/testGui/gestor.rb
require_relative 'cesar'
require_relative 'reverse'
require_relative 'reverseG'

class Gestor

    attr_accessor :palabra
    attr_accessor :cesar
    attr_accessor :reverse
    attr_accessor :reverse_g

  
    def initialize
      @palabra = "gest"
      @cesar = Cesar.new
      @reverse = Reverse.new
      @reverse_g = ReverseG.new
    end

    def cifrar( pala )
        @cesar.palabra = pala
        @reverse.palabra = pala
        @reverse_g.palabra = pala

        @cesar.cifrar()
        @reverse.cifrar()
        @reverse_g.cifrar()

    end

    def cifradoSinActualizacion
        @cesar.cifrar()
        @reverse.cifrar()
        @reverse_g.cifrar()
    end
end


